src/Microsoft.ML.DataView/DataViewRowId.cs

```diff
@@ -34,6 +34,16 @@ public DataViewRowId(ulong low, ulong high)
             High = high;
         }
 
+        public static bool operator ==(DataViewRowId left, DataViewRowId right)
+        {
+            return left.Equals(right);
+        }
+
+        public static bool operator !=(DataViewRowId left, DataViewRowId right)
+        {
+            return !(left == right);
+        }
```

Comment:
This fixes CA2231 on the ML.DataView project. This is a solution automatically generated by VisualStudio when looking into its suggestions on how to fix a given issue.